tests/Integration/Maker/MakeFactoryTest.php

```diff
@@ -34,6 +36,7 @@
  * @group maker
  */
 #[Group('maker')]
+#[RequiresPhp('>=8.4')]
```

Comment:
I skip those tests in other PHP versions, otherwise, we'd need to duplicate all the expected files, in order to only change the extended factory class